class AuthenticationError(Exception):
    """Raised when authentication fails"""
    pass